plugins/newsdownloader.koplugin/main.lua

```diff
@@ -116,7 +116,7 @@ function NewsDownloader:lazyInitialization()
         if news_downloader_settings:has(config_key_custom_dl_dir) then
             news_download_dir_path = news_downloader_settings:readSetting(config_key_custom_dl_dir)
         else
-            news_download_dir_path = ("%s/%s/"):format(DataStorage:getDataDir(), news_download_dir_name)
+            news_download_dir_path = ("%s/%s/"):format(DataStorage:getFullDataDir(), news_download_dir_name)
```

Comment:
You could also change it into a relative and an absolute path if you wish to maintain a short display in the Help/Settings info popup.

Reply:
Or rather a display path and a for actual use path.

Reply:
I think full path is even better. User wouldn't have to guess what it means

Reply:
Fine by me.